### ISSUE TITLE
Underlying Go compiler errors are formatted differently

### ISSUE THREAD

**Issue by alecthomas** [opennota/check#10]
Underlying errors in the source are reported in a different format to check itself. eg.

```
$ structcheck ./example
./example: example/stutter.go:26:6: a declared but not used
```

It would be good for automated tools if the errors were consistent.
